circle.yml

```diff
@@ -3,19 +3,26 @@ machine:
     LANG: en_US.UTF-8
     LC_CTYPE: en_US.UTF-8
   xcode:
-    version: "6.3.1"
+    version: "8.0"
 dependencies:
   pre:
-    - xcrun instruments -w "iPhone 6 (8.3 Simulator)" || exit 0
+    - xcrun instruments -w '23C6B49D-AF9A-4EBD-ABF3-A6960149327D' || true
   override:    
     - sudo gem install cocoapods xcpretty obcd -N
     - pod install --project-directory=Example
 test:
   override:
-    - set -o pipefail && xcodebuild -workspace Example/ExampleSSDataSources.xcworkspace
-      -scheme ExampleSSDataSources -sdk iphonesimulator -destination "platform=iOS Simulator,name=iPhone 6" 
-      GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=YES GCC_GENERATE_TEST_COVERAGE_FILES=YES clean test | xcpretty -c 
-      --report junit --output ${CIRCLE_TEST_REPORTS}/junit.xml
+    - set -o pipefail &&
+      xcodebuild
+        GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=YES 
+        GCC_GENERATE_TEST_COVERAGE_FILES=YES
+        -sdk iphonesimulator 
+        -destination 'platform=iOS Simulator,OS=9.0,name=iPhone 6'
+        -workspace Example/ExampleSSDataSources.xcworkspace
+        -scheme "ExampleSSDataSources"
+        clean build test |
+      tee $CIRCLE_ARTIFACTS/xcode_raw.log |
+      xcpretty --color --report junit --output $CIRCLE_TEST_REPORTS/xcode/results.xml
```

Comment:
Let's use the previous location for test output, please. This might also be a good time to update the test destination to iOS 10.